testing/meek/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# Contributor: kpcyrd <git@rxv.cc>
+# Maintainer: kpcyrd <git@rxv.cc>
+pkgname=meek
+pkgver=0.30
+pkgrel=0
+pkgdesc="A pluggable transport proxy written in Go"
+url="https://trac.torproject.org/projects/tor/wiki/doc/meek"
+arch="all"
+license="CC0 1.0 Universal"
```

Comment:
License format does not conform to https://spdx.org/licenses/
I think it should be "CC0-1.0".

Reply:
Thanks, done